Source/WebCore/PAL/pal/spi/cocoa/AVAssetWriterSPI.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020 Apple Inc. All rights reserved.
+ * Copyright (C) 2020-2025 Apple Inc. All rights reserved.
```

Comment:
I don't think that removing code counts as modifying code in a way that calls for copyright update. Unless you checked that each file was substantially modified, it might not be a correct change to make.

Reply:
Thanks for calling this out. I double checked and ended up removing it from a few files.